src/components/Adyen/Adyen.js

```diff
@@ -144,5 +144,4 @@ Adyen.defaultProps = {
 
 export { Adyen as PureAdyen };
```

Comment:
As Adyen also supports the translations we should add some keys, so there will be the possibility to change the Adyen copies. No needed for now, but could be added as a ticket to add in the future

Reply:
Good point. I've added a task for that.